lib/nimble_template/hex/package_mock.ex

```diff
@@ -0,0 +1,14 @@
+defmodule NimbleTemplate.Hex.PackageMock do
+  @moduledoc false
+
+  @behaviour NimbleTemplate.Hex.PackageBehaviour
+
+  alias NimbleTemplate.Exception.MockHexPackageRequiredException
+
+  def get_latest_version(package) do
```

Comment:
is it worth leaving a comment here, explaining the reason that we need to mock this function - to prevent the call to get the latest version if the test forgets to mock external library?

Reply:
added in https://github.com/nimblehq/elixir-templates/pull/284/commits/7a0aff5ce47ddb7c7245d660788fe44a6c8ffe76 @longnd